### PR TITLE
Prevent Meteors from knocking eachother out

### PR DESCRIPTION
**Pull request by ghost**
Meteors had a problem, made obvious by even the most cursory bhangmeter check, where most of the wave ended up wiping itself out through the following mechanism : 
- Two meteors bumped into eachother, thereby triggering eachother's Bump() proc and leading to an explosion
- Even an ex_act 3 removes meteors, causing the wave to be artificially thinned

This is notably the reason why the actual damage caused by increasing the number of Meteors in waves was bottlenecked, leading to the abject failure of the Meteors rework and a massive amount of unjustified lag

The following changes have been made to compound this problem : 
- Meteors can no longer be destroyed by ex_act(). While this may prevent insane Toxins workers from trying to ward off meteor waves by bombing them, this has the much more direct and useful effect of preventing a successful meteor hit from thinning out the entire wave
- Meteors now pass through eachother. Note that while bumping into eachother without exploding was considered, I cannot ensure it would work correctly, and it would otherwise need insane amounts of snowflake code to make sure it works correctly

Thus, the hit rate for meteor waves went from approximately 30 % to 100 %

Other changes : 
- Camera shake procs changed a bit
- Duplicate sounds removed (explosions already play explosion sounds, no need to double down on it)
- Formatting

Example of a new meaty-whores wave (Major Meteor Random Event) : 

100_percent_hitrate

### CODE REVIEW

**Comment by N3X15** [comment]
Travis fucked up when pulling updates, two builds completed without problem.  Approving despite travis

File: code/game/gamemodes/meteor/meteors.dm [hunk 1]
```diff
@@ -92,33 +92,40 @@
...
+/obj/effect/meteor/small
+	name = "small meteor"
+	icon_state = "smallf"
+	pass_flags = PASSTABLE
+
+/obj/effect/meteor/Move()
+	..()
 	return
 
 /obj/effect/meteor/small
```
Comment (xxalpha) [inline code/game/gamemodes/meteor/meteors.dm]: You forgot to delete what you copied from.

Comment (dylanstrategie) [inline code/game/gamemodes/meteor/meteors.dm]: Damnit

**Comment by PsiOmegaDelta** [comment]
I did some meteor tweaking on Bay earlier and remembered your explosion map. In case anyone is wondering why the east side of the map seems so unharmed, it may be due to the issue fixed here:
https://github.com/Baystation12/Baystation12/pull/11137

**Comment by dylanstrategie** [comment]
Actually, we have directional meteor waves, and I believe we have no problem with east-bound waves. I will have to doublecheck though

**Pull request merged**